com.github.SubhadeepJasu.pebbles.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: com.github.SubhadeepJasu.pebbles
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+base: io.elementary.BaseApp
+base-version: juno-20.08
+command: com.github.subhadeepjasu.pebbles
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=session-bus
```

Comment:
That's a bit too much, what exactly the app needs to talk to in the session bus? you can run the app with --log-session-bus to figure out which bus's it needs access to

Reply:
I've disabled the `session-bus` and run it with `--log-session-bus` however I can't interpret the output.

```
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.com.github.SubhadeepJasu.pebbles
C3: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for com.github.SubhadeepJasu.pebbles
C5: -> org.freedesktop.DBus fake ListNames
B1: <- org.freedesktop.DBus return from C1
B2: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B3: <- org.freedesktop.DBus return from C2
*SKIPPED*
B4: <- org.freedesktop.DBus return from C3
*SKIPPED*
B5: <- org.freedesktop.DBus return from C4
*SKIPPED*
B6: <- org.freedesktop.DBus return from C5
C6: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C7: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C8: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C10: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B7: <- org.freedesktop.DBus return from C6
*SKIPPED*
B8: <- org.freedesktop.DBus return from C7
*SKIPPED*
B9: <- org.freedesktop.DBus return from C8
*SKIPPED*
B10: <- org.freedesktop.DBus return from C9
*SKIPPED*
B11: <- org.freedesktop.DBus return from C10
*SKIPPED*
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
Filtering message due to arg0 com.github.subhadeepjasu.pebbles, policy: 0 (required 3)
*HIDDEN* (ping)
B12: <- (no sender) return from C11
*REWRITTEN*
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
This can be dropped now :)